src/RestWrite.js

```diff
@@ -353,14 +353,16 @@ RestWrite.prototype.transformUser = function() {
   if (this.query) {
```

Comment:
@bhaskaryasa 
Pls check the beginning of this method. There is an if statement like:
```
  if (this.className !== '_User') {
    return;
  }
```

Should this method return a promise?

I know this is not your code and it is the original code but I think we should correct it if it is incorrect. WDYT?